ee/codegen/src/context/workflow-context/workflow-context.ts

```diff
@@ -114,6 +114,9 @@ export class WorkflowContext {
   private readonly mlModelNamesById: Record<string, string> = {};
   private readonly errors: BaseCodegenError[] = [];
 
+  // Track generic node files for code merging
+  private readonly genericNodeFiles: Set<string> = new Set();
```

Comment:
[q] can we name this to something specific to needing python code merging rather than just generic node files? As mentioned, I think this is a set that we will look to decrease over time and drive most of it through codegen the more we enable editing in the UI